examples/python/ibmq/using_qiskit_terra_level_2.py

```diff
@@ -20,20 +20,11 @@
 # Import the Qiskit modules
 from qiskit import IBMQ, BasicAer
 from qiskit.circuit import QuantumCircuit
-from qiskit.circuit.library.standard_gates import SwapGate
-from qiskit.compiler import assemble
+from qiskit.compiler import transpile, assemble
 from qiskit.providers.ibmq import least_busy
 from qiskit.tools.monitor import job_monitor
 
-from qiskit.transpiler import PassManager
 from qiskit.transpiler import CouplingMap
-from qiskit.transpiler.passes import Unroller
-from qiskit.transpiler.passes import FullAncillaAllocation
-from qiskit.transpiler.passes import EnlargeWithAncilla
-from qiskit.transpiler.passes import TrivialLayout
-from qiskit.transpiler.passes import Decompose
-from qiskit.transpiler.passes import CXDirection
-from qiskit.transpiler.passes import LookaheadSwap
 
 
 provider = IBMQ.load_account()
```

Comment:
If a goal is to get these examples running in CI, maybe this could be converted to a `qiskit.test.mock.FakeProvider`.

Reply:
No opinion about running in CI. It would prevent drift, but it would also make the example less readable for someone to cut and paste from, which is presumably the reason it exists in the first place.

Reply:
We can probably play a trick here and inject the import and mock provider initialization at the beginning of the file like we do for jupyter executes in docstrings. That would let us test this in ci and keep usable examples in the repo.